core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1,78 +1,312 @@
+import software.amazon.awssdk.services.applicationautoscaling.model.TargetTrackingScalingPolicyConfiguration;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
+import software.amazon.awssdk.services.dynamodb.model.AttributeDefinition;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.CreateTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DeleteItemRequest;
+import software.amazon.awssdk.services.dynamodb.model.DeleteTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableResponse;
+import software.amazon.awssdk.services.dynamodb.model.DynamoDbException;
+import software.amazon.awssdk.services.dynamodb.model.GlobalSecondaryIndex;
+import software.amazon.awssdk.services.dynamodb.model.KeySchemaElement;
+import software.amazon.awssdk.services.dynamodb.model.KeyType;
+import software.amazon.awssdk.services.dynamodb.model.ListTablesResponse;
+import software.amazon.awssdk.services.dynamodb.model.LocalSecondaryIndex;
+import software.amazon.awssdk.services.dynamodb.model.PointInTimeRecoverySpecification;
+import software.amazon.awssdk.services.dynamodb.model.Projection;
+import software.amazon.awssdk.services.dynamodb.model.ProjectionType;
+import software.amazon.awssdk.services.dynamodb.model.ProvisionedThroughput;
+import software.amazon.awssdk.services.dynamodb.model.ResourceNotFoundException;
+import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType;
+import software.amazon.awssdk.services.dynamodb.model.ScanRequest;
+import software.amazon.awssdk.services.dynamodb.model.ScanResponse;
+import software.amazon.awssdk.services.dynamodb.model.TableStatus;
+import software.amazon.awssdk.services.dynamodb.model.UpdateContinuousBackupsRequest;
 
 @ThreadSafe
 public class DynamoAdmin implements DistributedStorageAdmin {
+  private static final Logger LOGGER = LoggerFactory.getLogger(DynamoAdmin.class);
+  private static final String PARTITION_KEY = "concatenatedPartitionKey";
+  private static final String CLUSTERING_KEY = "concatenatedClusteringKey";
+  private static final String GLOBAL_INDEX_NAME_PREFIX = "global_index";
+  private static final int CREATE_WAIT_DURATION_SECS = 3;
+  private static final int COOL_DOWN_DURATION_SECS = 60;
+  private static final double TARGET_USAGE_RATE = 70.0;
+  private static final int DELETE_BATCH_SIZE = 100;
+
+  public static final String NO_SCALING = "no-scaling";
+  public static final String NO_BACKUP = "no-backup";
+  public static final String REQUEST_UNIT = "ru";
+  public static final String DEFAULT_NO_SCALING = "false";
+  public static final String DEFAULT_NO_BACKUP = "false";
+  public static final String DEFAULT_RU = "10";
```

Comment:
I'm very sorry, one more thing. Can you please rename it to `DEFAULT_REQUEST_UNIT`:
```suggestion
  public static final String DEFAULT_REQUEST_UNIT = "10";
```

Reply:
Apply in [1d82dc8](https://github.com/scalar-labs/scalardb/commit/1d82dc8225ed711c766f30acad9e4764e081d40e)